### ISSUE TITLE
DC 2022 Annual not downloading from GetComics

### ISSUE THREAD

**Issue by SenorSmartyPants** [mylar3/mylar3#1059]
**Describe the bug**
Justice League 2022 Annual not downloading from GetComics.

**CarePackage**
[carepackage.zip](https://github.com/mylar3/mylar3/files/8063148/carepackage.zip)

**Environment (please complete the following information):**
 - OS: [e.g. Windows10]
 - Branch / commit: python3-dev

**Additional information**
[Justice League 2022 Annual on GC](https://getcomics.info/dc/justice-league-2022-annual-2022/)

Possibly relevant log entry
`Justice League 2022 Annual (2022) was not a match to Justice League 2022 Annual (2018)`

**Comment by evilhero** [comment]
This is proving tougher than normal for some reason - ``2021 Annual`` types worked fine, but the 2022 variations don't seem to be agreeing. Although at this point it seems to be isolated to just GC - in testing it was able to grab the 2022 version from a newznab provider just fine (which is the difference in web parsing vs API returns) as opposed to not matching up on GC.

Still working on it, but just might be a little bit before I can iron out the problems thus far...

**Comment by SenorSmartyPants** [comment]
Ran into this again with Action Comics Annual 2022 and getcomics.

https://getcomics.info/dc/action-comics-2022-annual/

https://comicvine.gamespot.com/action-comics-2022-annual/4050-143254/

Care Package
https://file.io/Od7dofQEKbuX

**Comment by evilhero** [comment]
Yeah I saw your forum post as well - it's inadvertently using the series year of the parent series when searching as opposed to the annual year itself (when you have annual integration enabled).

Currently looking at it now that I have a few moments to spare...

**Comment by SenorSmartyPants** [comment]
Still an issue with this weeks DC 2022 annuals.

**Comment by SenorSmartyPants** [comment]
This is still not working correctly. Linked is a log searching for Detective Comics 2022 Annual on GetComics

https://www.file.io/VgMt/download/MW3SVul5IswG

**Comment by claym** [comment]
> This is still not working correctly. Linked is a log searching for Detective Comics 2022 Annual on GetComics
> 
> https://www.file.io/VgMt/download/MW3SVul5IswG

+1 I can't get it to detect or post process after manually downloading

**Comment by barbequesauce** [comment]
Closing due to staleness, we welcome contributions for new features. If this is an issue and you are still experiencing it, please open a new ticket with a care package attached.
